Docs: Add Installation YouTube Videos to documentation pages. (#88443)
grafana/grafana: docs/sources/setup-grafana/installation/_index.md

To run Grafana, you must have a supported operating system, hardware that meets or exceeds minimum requirements, a supported database, and a supported browser.

> The following video guides you through the steps, common commands and the ways in which you can install Grafana in all of the operating systems described in this document:

Reviewer: ```suggestion
The following video guides you through the steps, common commands, and how to install Grafana in all of the operating systems described in this document:
```
